retext.py

```diff
@@ -54,6 +57,39 @@ def main():
 	app.setOrganizationDomain('mitya57.me')
 	if hasattr(app, 'setDesktopFileName'): # available since Qt 5.7
 		app.setDesktopFileName('me.mitya57.ReText.desktop')
+
+	singleApp = SingleApplication("D1278E7822F011E6802800F1F38F93EF")
```

Comment:
The number just an unique ID those all ReText shared, so that the new starting ReText will use to check if there already an ReText existed. It could be any text, but you must ensure it's unique value in system using by QSharedMemory (The key value of QSahredMemory).

The value in source code is generated by python code on my system:

`
import uuid
uuid.uuid1().hex.upper()
`